### PR TITLE
add font: lxgw-wenkai-v1.120

### PR DESCRIPTION
**Pull request by rtgiskard**
a  new beautiful Chinese font

the SRC_URI may be ready in few days

Signed-off-by: giskard <rtgiskard@gmail.com>

### CODE REVIEW

File: media-fonts/lxgw-wenkai/lxgw-wenkai-1.120.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+LICENSE="OFL-1.1"
+SLOT="0"
+KEYWORDS="amd64 ~arm64 ~ppc ~riscv x86 ~ppc-macos ~x64-macos"
+IUSE=""
+
+BDEPEND="$(unpacker_src_uri_depends)"
+
+# Only installs fonts
+RESTRICT="binchecks strip test"
```
Comment (ConiKost) [inline media-fonts/lxgw-wenkai/lxgw-wenkai-1.120.ebuild:23]: Why?

Comment (rtgiskard) [reply media-fonts/lxgw-wenkai/lxgw-wenkai-1.120.ebuild:23]: for the BDEPEND, as I have expected the tarball be tar.zst, this is support with unpacker.eclass, but looks like the upstream is do not provide tar.zst at the time, I'll revert back to tar.gz.

for RESTRICT, actually it is from another font ebuild, I'm not sure what it is, but several font ebuilds have it, so it's also copied here.

**Review by ConiKost — CHANGES_REQUESTED** [review]
Please fix CI error and drop commit for old version, as we only need for new version.
You can re-generate manifest with ```pkgdev manifest``` (dev-util/pkgdev) or ```repoman manifest``` (app-portage/repoman).

I also have to ask you to provide a proper full name for ```Signed-off-by```. Nick names are not valid.

File: media-fonts/lxgw-wenkai/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>rtgiskard@gmail.com</email>
+		<name>giskard</name>
```
Comment (ConiKost) [inline media-fonts/lxgw-wenkai/metadata.xml:6]: Please provide full name.

**Review by ConiKost — CHANGES_REQUESTED** [review]
lgtm. Just a very small nit pick: Could you update your name in the commit author field itself? Thanks!

**Comment by rtgiskard** [comment]
what do you mean by 'commit author' ? 👀

File: media-fonts/lxgw-wenkai/metadata.xml (hunk 2 above)
Comment (juippis) [reply media-fonts/lxgw-wenkai/metadata.xml]: This is not really required...but surely a nice thing.

File: media-fonts/lxgw-wenkai/lxgw-wenkai-1.200.ebuild [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+MY_P="${PN}-v${PV}"
+
+DESCRIPTION="An open-source Chinese font derived from Fontworks' Klee One"
+HOMEPAGE="https://github.com/lxgw/LxgwWenKai"
+SRC_URI="https://github.com/lxgw/LxgwWenKai/releases/download/v${PV}/${MY_P}.tar.gz"
+S=${WORKDIR}/${MY_P}
+
+LICENSE="OFL-1.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86"
```
Comment (juippis) [inline media-fonts/lxgw-wenkai/lxgw-wenkai-1.200.ebuild:17]: I'll drop to ~amd64 only. Please request keywording for other arches through bugzilla, if needed.

File: media-fonts/lxgw-wenkai/lxgw-wenkai-1.200.ebuild [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+EAPI=7
+
+inherit font
+
+MY_P="${PN}-v${PV}"
+
+DESCRIPTION="An open-source Chinese font derived from Fontworks' Klee One"
+HOMEPAGE="https://github.com/lxgw/LxgwWenKai"
+SRC_URI="https://github.com/lxgw/LxgwWenKai/releases/download/v${PV}/${MY_P}.tar.gz"
+S=${WORKDIR}/${MY_P}
```
Comment (juippis) [inline media-fonts/lxgw-wenkai/lxgw-wenkai-1.200.ebuild:13]:
```suggestion
S="${WORKDIR}/${MY_P}"
```

**Review by juippis — APPROVED** [review]

**Comment by juippis** [comment]
Damn this package seems to get updates every day!

**Comment by rtgiskard** [comment]
> Damn this package seems to get updates every day!

Take it easy, at least the author is active, and the font do looks pretty beautiful. I will watch for the major updates from the upstream to update the ebuild, Of course any update from others is also welcomed 😁

After so long, finally I have a PR done, I still remember the comments you made on my first attempt of the PR, that's really a long story 😉

**Comment by juippis** [comment]
Yeah, as you've seen, it may take many days merging stuff. It depends on how many other PRs are open, who are looking at these... but glad to hear you're only looking to ship major changes.

Baby steps, slow learning. Nice to find good quality PRs!

**Pull request closed without merging**